packages/consumption/src/modules/requests/outgoing/OutgoingRequestsController.ts

```diff
@@ -98,6 +99,8 @@ export class OutgoingRequestsController extends ConsumptionBaseController {
         parsedParams.content.id = id;
         const request = await this._create(id, parsedParams.content, parsedParams.peer);
 
+        await this._setDeletionInfo(parsedParams.content);
```

Comment:
Regarding your argumentation for restricting DeleteAttributeRequestItems to be sent via Messages, I can understand it very well. However, on the one hand, it would add a lot of validation complexity, as DeleteAttributeRequestItems may be used in the same Request alongside other RequestItems and I reckon this grows complex quickly. On the other hand, we can't guarantee that there are people whose usual flow is to exchange Requests by exchanging e.g. the truncated reference of a RelationshipTemplate with only the recipient and using the onExistingRelationship property. Of course this is not the desired behavior, as in this case rather Messages should be used. So, instead of catching the edge case of using a DeleteAttributeRequestItem in the onExistingRelationship property of a RelationshipTemplate, we could have a discussion about to which extent the onExistingRelationship property of RelationshipTemplates makes sense in general and should be used for. Maybe we don't want to exchange Requests using it altogether, but if you scan the RelationshipTemplate of a peer you already have a Relationship with, the respective Relationship is returned/displayed in the App. However, this exceeds the frame of this PR and we should probably discuss this elsewhere.

Reply:
I also think that this goes beyond the scope of this PR and that we need to think again about whether to allow the occurrence of the DeleteAttributeRequestItem in RelationshipTemplates. 👍 Perhaps this also fits a little bit into the topic of "unanswerable Requests", for which we have already created a ticket. So Requests that cannot be accepted depending on the Recipient should be able to be handled and displayed somehow. In general I consider the two properties `onNewRelationship` and `onExistingRelationship` of the RelationshipTemplateContent to be both important.

Reply:
I wrote a comment in the ticket named by you. 👍